### PR TITLE
Feature/add snippet

### PR DESCRIPTION
**Pull request by blimusiek**
added languages support

### CODE REVIEW

**Comment by vrael560** [comment]
@blimusiek Shouldn't we create table for holding languagues informations and fetch it for later use? I think now extending list of languages can be problematic

**Comment by blimusiek** [comment]
@vrael560 we can, but it will be a little problematic, because to add support for a new language we need to rebuild client app with js parser specific for that language. Currently I just pick most interesting languages for us and build it inside package.

We can also build inside our package all [languages](http://codemirror.net/3/mode/) but it will increase package size dramaticlly.

The other option is to load dynamically some languages but it will be in the other side a little problematic for ```webpack```

**Comment by vrael560** [comment]
@Vakrim

File: client/js/libs/languages.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  {mode: 'gfm', label: 'markdown', mime: 'text/x-markdown'},
+  {mode: 'haml', label: 'HAML', mime: 'text/x-haml'},
+  {mode: 'htmlmixed', label: 'HTML', mime: 'text/html'},
+  {mode: 'css', label: 'LESS', mime: 'text/x-less'},
+  {mode: 'javascript', label: 'JavaScript', mime: 'text/javascript'},
+  {mode: 'javascript', label: 'JSON', mime: 'application/json'},
+  {mode: 'nginx', label: 'Nginx', mime: 'text/nginx'},
+  {mode: 'ruby', label: 'Ruby', mime: 'text/x-ruby'},
+  {mode: 'sql', label: 'SQL', mime: 'text/x-sql'},
+  {mode: 'yaml', label: 'YAMP', mime: 'text/x-yaml'}
```
Comment (Vakrim) [inline client/js/libs/languages.js]: YAML

Comment (blimusiek) [inline client/js/libs/languages.js]: @Vakrim  done

**Pull request merged**